recipes/data-management/quantumleap/docker-compose.yml

```diff
@@ -0,0 +1,129 @@
+version: '3.3'
+
+services:
+
+  quantumleap:
+    image: smartsdk/quantumleap
+    ports:
+      - 8668:8668
+    depends_on:
+      - crate
+    environment:
+      - CRATE_HOST=crate
+    networks:
+      - backend
+      - frontend
+
+  traefik:
```

Comment:
I somehow agree, that's why I raised the discussion the other days on how we'll factor out the usage of traefik and how to document it. I'd rather do this in a different PR, because now this is the only recipe using traefik and without it it won't work.

Reply:
ok